integration-test/1106-merge-ocean-earth.py

```diff
@@ -0,0 +1,4 @@
+# There should be a single (merged) ocean feature in this tile
+assert_less_than_n_features(9, 167, 186, 'ocean', properties, 2)
+# There should be a single (merged) earth feature in this tile
+assert_less_than_n_features(9, 170, 186, 'earth', properties, 2)
```

Comment:
We should test once for OSM and once again for Natural Earh, so these tests again but for a zoom 5 coordinate.